backend/pkg/auth/user_registered_claims.go

```diff
@@ -5,4 +5,8 @@ import "github.com/golang-jwt/jwt/v4"
 type UserRegisteredClaims struct {
 	UserMetadata
 	jwt.RegisteredClaims
+
+	// Optional fields for access tokens
+	TokenID   string `json:"token_id,omitempty"`
+	TokenType string `json:"token_type,omitempty"`
```

Comment:
what's the intention here? Adding a tokenID to a JWT's token claims is kind of unusual